5 - RPG/Code/RPG.CharacterBuilder/CharacterAttributeClasses/FitnessLevel.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace RPG.CharacterBuilder.CharacterAttributeClasses
+{
+    public enum FitnessLevelEnum
+    {
+        ExtremelyFit,
+        VeryFit,
+        SomewhatFit,
+        NotFit,
+        FarFromFit
+    }
+    public class FitnessLevel : CharacterPropertyBase<FitnessLevelEnum>
+    {
+        private static FitnessLevelEnum _defaultValue = FitnessLevelEnum.SomewhatFit;
+        public override FitnessLevelEnum DefaultValue { get; internal set; } = _defaultValue;
+        public FitnessLevel() : base(_defaultValue)
+        {
+
+        }
+        public FitnessLevel(FitnessLevelEnum value) : base(value)
+        {
+
+        }
+
+        public override bool IsValid(FitnessLevelEnum value)
+        {
+            return true;
+        }
```

Comment:
You may want to re-think your base class design. It may be doing to much and to generic. When you see stuff like this, this is a sign of doing-to much when you are forced to override a member that has no place and or no need in the current design of the current class. 

To be clear, I am not saying not to use generics and not to use base class, but make sure they don't do "to much".

It is also possible that sometimes you simply don't use a given / known base class for its just not needed!